ui/app/styles/components/masked-input.scss

```diff
@@ -66,7 +66,6 @@
     height: auto;
     line-height: 1rem;
     min-width: $spacing-l;
```

Comment:
the icons were showing in front of the modal, we determined it was the z-index causing this.